package.json

```diff
@@ -16,7 +16,7 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.5",
+    "@mate-academy/scripts": "^0.9.7",
```

Comment:
Please do not add useless files to your PR.
You can use **git add src/** command to add needed files
**git status** command — for checking file status

Reply:
I do so, I don't understand what the problem is
![Screenshot_1](https://user-images.githubusercontent.com/72013210/140939746-b6f06245-275e-486b-b742-ea1d708b80c1.png)